bio/bedtools/complement/wrapper.py

```
@@ -7,13 +7,16 @@
 
 extra = snakemake.params.get("extra", "")
 
+compress = "| bgzip" if snakemake.output[0].endswith(".gz") else ""
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)